README.md

```diff
@@ -263,17 +263,49 @@ Onfido.init({
   containerId: 'onfido-mount',
   // here we send the data in the complete callback
   onComplete: function() {
-    var data = Onfido.getCaptures()
-    sendToServer(data)
+    var captures = Onfido.getCaptures()
+    sendToServer(captures)
   }
 })
 
-function sendToServer(data) {
-  var request = new XMLHttpRequest()
-  request.open('POST', '/your/server/endpoint', true)
-  request.setRequestHeader('Content-Type', 'application/json')
-  var dataString = JSON.stringify(data)
-  request.send(dataString)
+const reduceObj = (object, callback, initialValue) =>
```

Comment:
fair point on the separate PR.
Making it part of the SDK would not be a bad idea. What do you think @stefaniacardenas?